src/styles/index.scss

```diff
@@ -1,3 +1,244 @@
+  content: '';
+  position: absolute;
+  bottom: 0;
+  left: 0;
+  right: 0;
+  border-radius: 8px;
+  height: 4px;
+  background-color: #00acdc;
+}
+
+.container {
+  padding: 50px 40px;
+  display: grid;
+  grid-template-columns: repeat(1, 200px);
+  gap: 48px 46px;
+  justify-content: center;
+}
+
+@media (min-width: 488px) {
+  .container {
+    grid-template-columns: repeat(2, 200px);
+  }
+}
+
+@media (min-width: 768px) {
+  .container {
+    grid-template-columns: repeat(3, 200px);
+  }
+}
+
+@media (min-width: 1024px) {
+  .container {
+    grid-template-columns: repeat(4, 200px);
+  }
+}
+
+.card {
+  margin: auto;
+  width: 200px;
+  height: 408px;
+  border: 1px solid #f3f3f3;
+  padding-top: 32px;
+  box-sizing: border-box;
+  transition: transform 300ms;
+
+  &__title {
+    font-size: 12px;
+    font-weight: 500;
+    line-height: 18px;
+    text-align: left;
+    margin: auto;
+    color: rgba(6, 11, 53, 1);
+    padding: 40px 16px 0;
+
+    &-p {
+      padding: 0;
+      margin: 0;
+    }
+  }
+
+  &:hover {
+    transform: scale(1.2);
+
+    .card__title {
+      color: #34568b;
+    }
+  }
+
+  &__image {
+    background-image: url(../images/imac.jpeg);
+    background-size: 100% 100%;
+    width: 160px;
+    height: 134px;
+    margin: auto;
+    border: none;
+    padding: 0;
+  }
+
+  &__code {
+    font-size: 10px;
+    font-weight: 400;
+    line-height: 14px;
+    text-align: left;
+    margin: auto;
+    color: #616070;
+    padding: 4px 16px 0;
+
+    &-p {
+      padding: 0;
+      margin: 0;
+    }
+  }
+
+  &-stars {
+    display: flex;
+  }
+
+  &-stars__star {
+    background-image: url(../images/star.svg);
+    height: 16px;
+    width: 16px;
+    background-repeat: no-repeat;
+    background-position: center;
+    margin: 0 4px 0 0;
+  }
+
+  &-stars__star:not(:last-child) {
+    padding: 0;
+  }
+
+  &-stars--4 &-stars__star:nth-child(-n + 4) {
+    background-image: url(../images/star-active.svg);
+  }
+
+  &__rate {
+    display: flex;
+    padding: 16px 16px 0;
+    justify-content: space-between;
+
+    &-review {
+      font-family: Roboto, sans-serif;
+      font-size: 10px;
+      font-weight: 400;
+      line-height: 14px;
+      vertical-align: bottom;
+      text-align: right;
+
+      &-p {
+        margin: 0;
+        padding: 2px 0 0;
+      }
+    }
+  }
+
+  &__price {
+    display: flex;
+    justify-content: space-between;
+    padding: 24px 16px 0;
+
+    &-text {
+      font-family: Roboto, sans-serif;
+      font-size: 12px;
+      font-weight: 400;
+      line-height: 18px;
+      text-align: left;
+      text-underline-position: from-font;
+      text-decoration-skip-ink: none;
+      color: #616070;
+      padding: 0;
+      margin: 0;
+    }
+
+    &-numb {
+      font-family: Roboto, sans-serif;
+      font-size: 16px;
+      font-weight: 700;
+      line-height: 18px;
+      text-align: right;
+      text-underline-position: from-font;
+      text-decoration-skip-ink: none;
```

Comment:
The 'text-decoration-skip-ink' property is not supported in all browsers. Consider using a fallback or checking browser compatibility to ensure consistent appearance.